lambdas/importer.ts

```diff
@@ -7,7 +7,13 @@ async function fetchEvents(meetupAccessToken: string) {
 	const meetupGraphQlEndpoint = 'https://api.meetup.com/gql';
 	const batchSize = 10; // Number of events to fetch in each batch
 
-	const GROUP_URLNAMES = ['open-sgf', 'springfield-women-in-tech', 'sgfdevs'];
+	const GROUP_URLNAMES = [
+		'open-sgf',
+		'springfield-women-in-tech',
+		'sgfdevs',
+		'sgfaws',
+		'sgfdotnet',
+	];
```

Comment:
Can we move these values to an environment variable?

Side note: arrays are normally not a valid data type in environment variables. What I do in that case is make it a single string with the values separated by commas. Then I'll use the js `split` function to split that string int an array on the comma characters

Reply:
I moved GROUP_URLNAMES to the .env.example file and split the string like you said.

I also had to throw an error in case GROUP_URLNAMES is undefined to make TypeScript happy.